app/components/avo/fields/common/key_value_component.rb

```diff
@@ -3,11 +3,10 @@
 class Avo::Fields::Common::KeyValueComponent < Avo::BaseComponent
   include Avo::ApplicationHelper
 
-  attr_reader :view
-
-  def initialize(field:, form: nil, view: "show")
-    @field = field
-    @form = form
-    @view = Avo::ViewInquirer.new(view)
+  prop :field, Avo::Fields::BaseField
+  prop :form, _Nilable(ActionView::Helpers::FormBuilder)
+  prop :view, Avo::ViewInquirer do |value|
+    value = :show if value.nil?
```

Comment:
Can we use `default` here?

```suggestion
  prop :view, Avo::ViewInquirer, default: :show do |value|
    value = :show if value.nil?
```